docs/security.md

```diff
@@ -37,26 +40,35 @@ Both stores must be base64 encoded, for example:
 
 **Note:** The base64 string of the keystore will probably be much longer than the snippet above, spanning 50 lines or so.
 
-With this and the password `secret` for the keystore and the private key, your JSON options file will look like this:
+Add the stores to your secrets in the DC/OS Secret store, for example if your base64 encoded keystores and truststores are server.jks.base64 and trust.jks.base64, respectively then do the following: 
+
+```bash
+dcos security secrets create /truststore --value-file trust.jks.base64
+dcos security secrets create /keystore --value-file server.jks.base64
+```
 
-    {
-      "security": {
-        "ssl": {
-          "enabled": true,
-          "keyStoreBase64": "/u3+7QAAAAIAAAACAAAAAgA...”,
-          "keyStorePassword": "secret",
-          "keyPassword": "secret"
-        }
-      }
-    }
+In this case you're adding two secrets `/truststore` and `/keystore` that you will need to pass to the Spark Driver and Executors. You will need to add the following configurations to your `dcos spark run ` command:
 
-Install Spark with your custom configuration:
+```bash
 
-    dcos package install --options=options.json spark
+dcos spark run --verbose --submit-args="\
+--conf spark.mesos.containerizer=mesos \  # use mesos containerizer
+--conf spark.ssl.enabled=true \
+--conf spark.ssl.enabledAlgorithms=TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA \
+--conf spark.ssl.keyPassword=<key password> \
+--conf spark.ssl.keyStore=server.jks \  # This MUST be set this way
+--conf spark.ssl.keyStorePassword=<keystore access password> \
+--conf spark.ssl.protocol=TLS \
+--conf spark.ssl.trustStore=trust.jks \  # this MUST be set this way
+--conf spark.ssl.trustStorePassword=<truststore password> \
+--conf spark.mesos.driver.labels=DCOS_SECRETS_DIRECTIVE:[{\"name\"\:\"/keystore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"KEYSTORE_BASE64\"}}\,{\"name\"\:\"/truststore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"TRUSTSTORE_BASE64\"}}] \
+--conf spark.mesos.task.labels=DCOS_SECRETS_DIRECTIVE:[{\"name\"\:\"/keystore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"KEYSTORE_BASE64\"}}\,{\"name\"\:\"/truststore\"\,\"type\"\:\"ENVIRONMENT\"\,\"environment\"\:{\"name\"\:\"TRUSTSTORE_BASE64\"}}],DCOS_SPACE:/spark, \
```

Comment:
This is a pretty long command. Did you consider setting some of these configs automatically in the CLI? Or, do you want the user to be aware of each config being set?